test/functional/test_framework/key.py

```diff
@@ -356,6 +356,26 @@ def tweak_add(self, tweak):
         ret.compressed = self.compressed
         return ret
 
+    def mul(self, data):
+        assert(self.valid)
+        assert(len(data) == 32)
+        f = int.from_bytes(data, 'big')
+        assert(f < SECP256K1_ORDER and f is not None)
+        ret = ECPubKey()
+        ret.p = SECP256K1.mul([(self.p, f)])
+        ret.valid = True
+        ret.compressed = self.compressed
+        return ret
+
+    def add(self, pk):
```

Comment:
Add a docstring commenting this function (ie that this adds two pubkeys)

Reply:
Done.